cmd/mesh/testdata/manifest-generate/output/ingressgateway_k8s_settings.yaml

```diff
@@ -0,0 +1,6225 @@
+# Resources for Base component
```

Comment:
Could you delete anything that's not being compared from the output? It looks like only Deployment and Service are tested. Just keep those to make the output readable.

Reply:
yep, removed